todo_app/app.py

```diff
@@ -1,33 +1,100 @@
 from flask import Flask, render_template, request
+from flask_login import LoginManager, current_user, login_required, login_user
+import os
+import requests
 from werkzeug.utils import redirect
 from todo_app.data.mongodb_items import get_items, create_item, delete_item, change_item_status
 from todo_app.flask_config import Config
+from todo_app.models.User import User
 from todo_app.models.ViewModel import ViewModel
+from functools import wraps
+from flask import abort
 
 def create_app():
     app = Flask(__name__)
     app.config.from_object(Config())
+    login_manager = LoginManager()
+    app.config['LOGIN_DISABLED'] = os.getenv('LOGIN_DISABLED') == 'True'
+
+    auth_mapping = {"78739678": "writer"}
+
+    def user_in_group(current_user, group):
+        if(app.config.get('LOGIN_DISABLED') == True):
+            return True
+        if current_user is None or not current_user.is_authenticated:
+            return False
+        if auth_mapping[current_user.get_id()] != group:
```

Comment:
This key-read throws an error if the key doesn't exist - is that what you want? That's a legitimate design, but worth noting explicitly that this doesn't set users to readers by default which was our working assumption